osmo-test-5/osmosis.zone_assets.json

```diff
@@ -110,6 +110,12 @@
       "base_denom": "aconst",
       "path": "transfer/channel-7779/aconst",
       "osmosis_verified": false
+    },
+    {
+      "chain_name": "symphonytestnet",
+      "base_denom": "note",
+      "path": "transfer/channel-8686/note",
```

Comment:
This IBC connection isn't registered at the Chain Registry

Reply:
this is on testnet.  is there something wrong with this registration?

https://github.com/cosmos/chain-registry/blob/master/testnets/_IBC/osmosistestnet-symphonytestnet.json

Reply:
My mistake. It is there. I'll need to figure out why the validation says that it's missing, then.

Reply:
In the registered IBC connection, there's 3 'n's in the word 'channel'.
https://github.com/cosmos/chain-registry/blob/master/testnets/_IBC/osmosistestnet-symphonytestnet.json#L16

Reply:
I've corrected the error at the CCR and added validation to ensure it can't happen again. The check passes now.